torch/_inductor/codegen/wrapper.py

```diff
@@ -1286,6 +1326,31 @@ def g(args):
             """
         )
 
+    def generate_c_shim_extern_kernel_call(self, kernel, args):
+        # In the abi_compatible mode, we call fallback aten ops through a C shim layer
+        kernel = "aoti_torch_" + kernel.split("::")[-1]
+        for i, arg in enumerate(args):
+            if arg.startswith("create_raii_tensor_handle_for_temp("):
```

Comment:
meh

Reply:
You can avoid needing to do the explicit `.get()` call by adding an implicit conversion to your RAII class, `operator AtenTensorHandle()`